src/Sarif.Multitool/FileWorkItemsCommand.cs

```diff
@@ -2,15 +2,105 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
+using System.Globalization;
+using System.Linq;
 
 namespace Microsoft.CodeAnalysis.Sarif.Multitool
 {
     public class FileWorkItemsCommand : CommandBase
     {
-        public int Run(FileWorkItemsOptions fileWorkItemsOptions)
+        private static readonly string[] s_knownFilingTargetTypes = new[] { "github", "azuredevops" };
+        private static readonly string[] s_knownFilteringStrategies = new[] { "new", "all" };
+        private static readonly string[] s_knownGroupingStrategies = new[] { "perresult" };
```

Comment:
You will definitely want enums for these, I think. :)

Reply:
Will do later. Not as important as demonstrating bug filing this week.

---
In reply to: [306575692](https://github.com/microsoft/sarif-sdk/pull/1583#discussion_r306575692) [](ancestors = 306575692)